.github/workflows/Release-candidate-containers.yml

```diff
@@ -0,0 +1,136 @@
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+  prepare-modernization-api-release-candidate:
+    name: Prepare Modernization API Candidate Deployment
+    needs: build-modernization-api-release-candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Update-helm-charts.yaml@main
+    with:
+      microservice_name: modernization-api
+      values_file_with_path: charts/modernization-api/values_${{inputs.values}}.yaml
+      new_image_tag: ${{ needs.call-build-microservice-container-workflow.outputs.output_image_tag }}
+    secrets:
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}      
+  build-elasticsearch-release-candidate:
+    if: ${{ inputs.build-elasticsearch == 'true' }}
+    name: Build Elasticsearch Release Candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Release-gradle-microservice-container.yaml@main
+    with:
+      microservice_name: elasticsearch    
+      build-new-container: true
+      dockerfile_relative_path: -f ./cdc-sandbox/elasticsearch/Dockerfile .
+      environment_classifier: ${{inputs.environment}}
+    secrets:
+      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
+      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+  prepare-elasticsearch-release-candidate:
+    name: Prepare Elasticsearch Release Candidate Deployment
+    needs: build-elasticsearch-release-candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Update-helm-charts.yaml@main
+    with:
+      microservice_name: elasticsearch
+      values_file_with_path: charts/elasticsearch-efs/values_${{inputs.values}}.yaml
+      new_image_tag: ${{ needs.call-build-microservice-container-workflow.outputs.output_image_tag }}
+    secrets:
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}        
+  build-nifi-release-candidate:
+    if: ${{ inputs.build-nifi == 'true' }}
+    name: Build NiFi Release Candidate
+    uses: CDCgov/NEDSS-Workflows/.github/workflows/Release-gradle-microservice-container.yaml@main
+    with:
+      microservice_name: nifi    
+      build-new-container: true
+      dockerfile_relative_path: -f ./cdc-sandbox/nifi/Dockerfile .
+      environment_classifier: ${{inputs.environment}}
+    secrets:
+      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
+      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
+      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
+      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
+      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+  prepare-nifi-release-candidate:
+    name: Prepare NiFi Release Candidate Deployment
```

Comment:
This needs the same if statement as line 111 or it will fail the pipeline.

Reply:
I stand corrected based on definition of the needs block.
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds